sonar-php-plugin/src/main/java/org/sonar/plugins/php/reports/phpunit/CoverageResultImporter.java

```diff
@@ -216,4 +111,30 @@ public String reportName() {
   public Logger logger() {
     return LOG;
   }
+
+  /**
+   * Small class that save Coverage for each found fileNode and keep track of count.
+   */
+  private static class CoverageMeasureRecorder implements Consumer<FileNode> {
+
+    private final CoverageResultImporter importer;
+    SensorContext context;
+    int fileNodeCount=0;
+
+    public CoverageMeasureRecorder(CoverageResultImporter importer, SensorContext context) {
+      this.importer = importer;
+      this.context = context;
+    }
+
+    public int getFileNodeCount() {
+      return fileNodeCount;
+    }
+
```

Comment:
I would remove this method and just access the field directly, since it is used only in one point in this same file
```suggestion
```